lib/dokumi/support/benchmark.rb

```diff
@@ -0,0 +1,55 @@
+require 'benchmark'
+require 'json'
+
+module Dokumi
+  module Support
+    class ExecutionLog
+      attr_reader :command, :tms, :start_time
+
+      def initialize(command, tms, start_time)
+        @command = command
+        @tms = tms
+        @start_time = start_time
+      end
+
+      def to_hash
+        { command: command,
```

Comment:
I think what Kazuaki was suggesting looked better, but if it's the official style... :(